.travis.yml

```diff
@@ -35,7 +36,11 @@ before_script:
   - psql -c 'grant all privileges on database simplified_circulation_test to simplified_test;' -U postgres
 
 env:
-  - SIMPLIFIED_TEST_DATABASE="postgres://simplified_test:test@localhost:5432/simplified_circulation_test"
+  global:
+    - SIMPLIFIED_TEST_DATABASE="postgres://simplified_test:test@localhost:5432/simplified_circulation_test"
+  matrix:
+    - SIMPLIFIED_ELASTICSEARCH_VERSION=1
+    - SIMPLIFIED_ELASTICSEARCH_VERSION=6
```

Comment:
On Slack you explained to me that this runs the test suite in parallel with both ES1 and ES6. This is a great idea, but the tests that exercise this dependency are in server_core, not circulation.

Everything else in this PR looks good -- upgrading PIP and installing from a different requirements file based on the variable name, and there's commented-out code in the core `travis.yml` for installing one version or another of Elasticsearch. Can you create a similar branch in core, and then remove this bit from the circulation PR?